packages/ui/src/components/ResourceBlock/ResourceBlock.tsx

```diff
@@ -0,0 +1,197 @@
+import { cloneElement, ReactElement, ReactNode } from "react";
+import CheckCircleIcon from "@heroicons/react/20/solid/CheckCircleIcon";
+import ExclamationCircleIcon from "@heroicons/react/20/solid/ExclamationCircleIcon";
+import ArrowRightIcon from "@heroicons/react/20/solid/ArrowRightIcon";
+
+import { HeadingLevel } from "../Heading";
+import { headingStyles } from "../Heading/styles";
+import { LinkProps } from "../Link";
+import { HydrogenAttributes } from "../../types";
+
+export const colorOptions = [
+  "primary",
+  "secondary",
+  "tertiary",
+  "quaternary",
+  "quinary",
+] as const;
+
+type CardColor = (typeof colorOptions)[number];
+
+const headingBarStyleMap: Record<CardColor, Record<string, string>> = {
+  primary: {
+    "data-h2-background-color": "base(primary.lightest)",
+    "data-h2-border-bottom": "base(primary.darkest)",
+  },
+  secondary: {
+    "data-h2-background-color": "base(secondary.lightest)",
+    "data-h2-border-bottom": "base(secondary.darkest)",
+  },
+  tertiary: {
+    "data-h2-background-color": "base(tertiary.lightest)",
+    "data-h2-border-bottom": "base(tertiary.darkest)",
+  },
+  quaternary: {
+    "data-h2-background-color": "base(quaternary.lightest)",
+    "data-h2-border-bottom": "base(quaternary.darkest)",
+  },
+  quinary: {
+    "data-h2-background-color": "base(quinary.lightest)",
+    "data-h2-border-bottom": "base(quinary.darkest)",
+  },
+};
+
+const wrapperStyleMap: Record<CardColor, Record<string, string>> = {
+  primary: {
+    "data-h2-color": "base(primary.darkest)",
+  },
+  secondary: {
+    "data-h2-color": "base(secondary.darkest)",
+  },
+  tertiary: {
+    "data-h2-color": "base(tertiary.darkest)",
+  },
+  quaternary: {
+    "data-h2-color": "base(quaternary.darkest)",
+  },
+  quinary: {
+    "data-h2-color": "base(quinary.darkest)",
+  },
+};
+
+interface RootProps {
+  title: ReactNode;
+  headingColor?: CardColor;
+  headingAs?: HeadingLevel;
+  children: ReactElement<ItemProps> | Array<ReactElement<ItemProps>>; // Restricts children to only expected items;
+}
+
+const Root = ({
+  title,
+  headingColor = "primary",
+  headingAs = "h3",
+  children,
+}: RootProps) => {
+  // prepare heading text element
+  const HeadingTextElement = headingAs;
+  const commonHeadingTextStyles = { ...headingStyles["h4"] };
+  delete commonHeadingTextStyles["data-h2-margin"];
+  return (
+    <div
+      data-h2-shadow="base(larger)"
+      data-h2-border-radius="base(rounded)"
+      data-h2-background-color="base(foreground)"
+    >
+      {/* heading bar */}
+      <div
+        {...headingBarStyleMap[headingColor]}
+        data-h2-border-radius="base(rounded rounded 0 0)"
+        data-h2-border-bottom-width="base(1px)"
+        data-h2-border-bottom-style="base(solid)"
+        data-h2-padding="base(x1) p-tablet(x1 x1.5 x1 x1.5)"
```

Comment:
This can be shortened to:
```css
data-h2-border-bottom="base(1px solid)"
data-h2-padding="base(x1) p-tablet(x1 x1.5)"
```

Reply:
Good catch.
[styles more terse](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11200/commits/d79b516d023d69616768a08aa8d28dfa6d4cab2e)